io-engine/src/bin/io-engine-client/v1/stats_cli.rs

```diff
@@ -124,6 +134,93 @@ async fn pool(mut ctx: Context, _matches: &ArgMatches) -> crate::Result<()> {
     Ok(())
 }
 
+async fn nexus(mut ctx: Context, matches: &ArgMatches) -> crate::Result<()> {
+    ctx.v2("Requesting Nexus metrics");
+    let n = matches.get_one::<String>("name");
```

Comment:
hmm might be good to also have an optional uuid to list the stats?

Reply:
@tiagolobocastro , Since that would require protobuf changes lets have it as part of different task.